### PR TITLE
Add downloads page

### PR DESCRIPTION
**Pull request by peatroot**
See https://www.targetvalidation.org/downloads/data.

### CODE REVIEW

**Comment by peatroot** [comment]
Screenshot 2019-09-24 at 12 30 35

File: src/public/downloads/Page.js [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+        <br />
+        <Typography>
+          All data from targetvalidation.org is available for download as
+          compressed JSON files.
+          <br />
+          <br />
+          We provide downloads of all associations between targets and diseases
+          calculated by the platform, as well as all the evidence used in
+          calculating each association. These are the same objects returned by
+          the corresponding <tt>/public/associations</tt> and{' '}
```
Comment (mirandaio) [inline src/public/downloads/Page.js]: First time I've encountered this `<tt>`. Looks like it is obsolete now according to this https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt. Also, HTML should be used for structure, not for styling.

Comment (peatroot) [reply src/public/downloads/Page.js]: Good catch. Following suggestion on link, I've swapped `tt` tags for `code` tags.

Comment (peatroot) [reply src/public/downloads/Page.js]: I also removed `br` tags and added the `paragraph={true}` prop on `Typography` components.

**Pull request merged**